modules/loader-utils/src/worker-utils/version.js

```diff
@@ -0,0 +1,31 @@
+// Returns `true` if the two versions are compatible
+export function validateLoaderVersion(coreVersion, loader) {
```

Comment:
Why do we need to validate both in core and loaders?

Reply:
The worker loaders do not use core.